javascript/boilerplate-text/boilerplate-fr.js

```diff
@@ -113,6 +114,7 @@ s.validXHTML = "XHTML 1.0 valide !";
 s.validCSS = "CSS valide !";
 s.codedInUtf8 = "Codé en UTF-8 !";
 
+s.translatedFromEnglishVer = "Traduit d’un contenu en anglais daté du "+dt.enVersion+". Dernière modification de cette traduction le "+dt.thisVersionPlain+" UTC."; 
```

Comment:
Not sure why this line is here. The translation already exists at line 126 of your version.